core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.ts

```diff
@@ -406,22 +412,32 @@ export class TemplateBuilderComponent implements OnInit, AfterViewInit, OnDestro
     }
 
     /**
-     * @description This method sets the box initial values everytime a mouse leaves a row
+     * @description This method sets the box initial values everytime a mouse moves in the grid
      * so that way we always have the correct value setted and overriding the behavior of gridstack
      *
      * @private
-     * @param {Element} el
      * @memberof TemplateBuilderComponent
      */
-    private fixGridstackNodeOnMouseLeave(el: Element): void {
-        // So every time the mouse leaves the row, we set the initial values for the box
-        el.addEventListener('mouseleave', () => {
-            if (this.addBoxIsDragging && this.addBox.nativeElement.gridstackNode?.w !== BOX_WIDTH) {
-                this.addBox.nativeElement.gridstackNode = {
-                    ...this.addBox.nativeElement.gridstackNode,
-                    ...this.boxOptions
-                };
-            }
-        });
+    private listenToGridMouseMove(): void {
+        // So every time the mouse moves in the grid, we set the initial values for the box
+        this.main.nativeElement.addEventListener(
+            'mousemove',
+            this.fixGridStackNodeOptions.bind(this)
+        );
```

Comment:
Since this function is triggered every time the mouse moves, it seems quite frequent. Is there any alternative approach we can explore to optimize this behavior?